app/models/comment.rb

```diff
@@ -1,2 +1,10 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_comments_counter
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] [OPTIONAL] Instead of calling count on records (image 1000 records, it will slow the application), we can simply use the ...increment... Ruby method. Something like this: 
```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```
__Since you are using an after_save callback function, this function should be a private function.__